src/components/runtime-common.js

```diff
@@ -124,7 +124,10 @@ export const SaveFilesHelp = () => {
 }
 
 export const GalaxyWarning = () => {
-  return p([div({ style: { fontWeight: 600 } }, ['Important: Please keep this tab open and logged in to Terra while using Galaxy.']), ' Galaxy will open in a new tab. '])
+  return div({ style: { fontWeight: 600 } },
```

Comment:
This is invalid html I found when digging around. <p> tags can have no children but text.

Reply:
A `p` tag should be able to have any children: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Also note that a paragraph element has margins applied to it automatically, but a div does not. You may wish to apply the font weight style to the p itself and remove the div.